nexus/src/app/background/tasks/support_bundle_collector.rs

```diff
@@ -780,11 +780,8 @@ where
     match result {
         Ok(result) => {
             let output = result.into_inner();
-            tokio::fs::write(
-                path.join(format!("{command}.txt")),
-                format!("{output:?}"),
-            )
-            .await?;
+            let json = serde_json::to_string(&output)?;
+            tokio::fs::write(path.join(format!("{command}.txt")), json).await?;
```

Comment:
Should we use a `.json` suffix here?

Related: Do we want the error types to be serialized to JSON too?

Reply:
Fixed in [e610f80](https://github.com/oxidecomputer/omicron/pull/7462/commits/e610f804331a124f840cc2ad477c05b4e52f9537)

I am inclined to leave the error file alone as I feel like most of the structured output parsing will happen on the "success" path where there may be multiple commands that have ran with output that you would like to filter through.